ui/app/components/clients/charts/vertical-bar-basic.ts

```diff
@@ -52,40 +50,48 @@ export default class ClientsSyncBarChartComponent extends Component<Args> {
 
   get chartData() {
     return this.args.data.map((d): ChartData => {
-      const date = parseAPITimestamp(d.timestamp) as Date;
```

Comment:
I didn't see the need to change this to a `Date` object because below we format it back from a date to `M/yy`. Plus, `scaleBand` which is used to build the x-axis just spaces the values evenly based on how many exist. Dates are already sorted by the backend.